src/Hungary/Mews.Fiscalizations.Hungary/RequestCreator.cs

```diff
@@ -61,25 +86,34 @@ private static Dto.ManageInvoiceRequest CreateManageInvoicesRequest<T>(Technical
         }
 
         private static T CreateRequest<T>(TechnicalUser user, SoftwareIdentification software, string additionalSignatureData = null)
-            where T : Dto.BasicRequestType, new()
+            where T : Dto.BasicOnlineInvoiceRequestType, new()
         {
             var requestId = RequestId.CreateRandom();
-            var timestamp = DateTime.UtcNow;
+            var nowUtc = DateTime.UtcNow;
+            var timestamp = new DateTime(nowUtc.Year, nowUtc.Month, nowUtc.Day, nowUtc.Hour, nowUtc.Minute, nowUtc.Second, DateTimeKind.Utc);
```

Comment:
I suppose this is about dropping the information about milliseconds?

Reply:
Yup, otherwise it would fail :D they have this validation on their end.